### DISCUSSION TITLE
Override the column name when logging activity?

### DISCUSSION THREAD

**Discussion by airliajsmith** [spatie/laravel-activitylog#791] [Q&A]
When I log a change to a Sales Order, it comes back with the name of the column 'number', but I want to display 'Sales Order Number'. Is there a way to change that before it's logged?

**Comment by Gummibeer** [comment]
You could use an observer or the `tapActivity()` method on your order model.
But I highly recommend to do it in your template. Because you should already have this mapping of `order.number` to your display label for forms or your read view.
And if somewhen this message changes or you want to translate it you will have an easier job to do so by using the real attribute name plus related model instead of multiple already, partial, customized/translated strings.

**Reply by airliajsmith** [reply]
I can't find where mapping is mentioned in the documentation. Can you post a link?

**Reply by Gummibeer** [reply]
There is no paragraph related to mapping because there are so many ways to do it.
One way would be for example to use the already translated field names and append your own in the `validation` translation file.

**Reply by airliajsmith** [reply]
Sorry, I'm still quite a newbie. Can you point to some website article that could explain the validation translation file or anything regarding mapping? I'm not familiar with either of those and I'm completely lost.
